packages/json-schema/test/integration/build-schema.test.ts

```diff
@@ -0,0 +1,215 @@
+// Copyright IBM Corp. 2013,2018. All Rights Reserved.
+// Node module: @loopback/openapi-spec-builder
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {model, property, ModelMetadataHelper} from '@loopback/repository';
+import {modelToJsonDef, toJsonProperty} from '../../src/build-schema';
+import {expect} from '@loopback/testlab';
+
+describe('build-schema', () => {
+  context('exception', () => {
```

Comment:
I think the usage of `contex` is correct here, see http://loopback.io/doc/en/contrib/style-guide.html#test-block-naming

In this particular case, I feel this particular nesting block is redundant and the test `it('errors out when "@property.array" is not used on an array'` should be moved into `describe('modelToJsonDef')` block below, because `modelToJsonDef` is the function executed by this test.